components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/IdentityClaimManager.java

```diff
@@ -91,35 +91,30 @@ public static IdentityClaimManager getInstance() throws IdentityException {
      */
     public Claim[] getAllSupportedClaims(String dialectUri, UserRealm realm)
             throws IdentityException {
-        Claim[] claims = new Claim[0];
+
         ClaimManager claimAdmin;
-        ArrayList<Claim> requiredClaims = null;
+        ArrayList<Claim> requiredClaims = new ArrayList<Claim>();
```

Comment:
Better to use List interface when creating variable and use diamond operator when initializing variable.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/1303